09-gis.Rmd

```diff
@@ -240,11 +239,11 @@ get_usage(alg)
 #>	...
 ```
 
-Conveniently, the user has not to specify each single parameter.
-In case a parameter is left unspecified, `run_qgis()` will automatically use the corresponding default value as argument if available.
+Conveniently, the user does not need to specify each single parameter.
+In case a parameter is left unspecified, `run_qgis()` will automatically use the corresponding default value as an argument if available.
 To find out about the default values, run `get_args_man()`.  
 
-To finally get rid off the slivers, we specify that all polygons with an area less or equal to 25000 m^2^ should be joined to that neighboring polygon with the largest area (see right panel of Figure \@ref(fig:sliver-fig)).
+To get rid off the slivers, we specify that all polygons with an area less or equal to 25000 m^2^ should be joined to the neighboring polygon with the largest area (see right panel of Figure \@ref(fig:sliver-fig)).
```

Comment:
suggestion: `to remove the slivers` (off is a typo)

Reply:
ok, changed! Thanks!